src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
+        int indexRandom = random.nextInt(Colors.values().length);
+        Colors colors = Colors.values()[indexRandom];
+        return String.valueOf(colors);
```

Comment:
check common mistakes before making PR:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants